server/controllers/user.controller.js

```diff
@@ -810,6 +812,73 @@ const UserController = function () {
     }
   };
 
+  var getPlayerDataByUserSessionId = async function (req, res) {
+    const id = req.params.user_session_id;
+    if (!id || isNaN(Number(id))) {
+      return res.status(400).json({
+        success: false,
+        error: `invalid value for id`,
+      });
+    }
+
+    try {
+      let data = await UserSessions.findOne({
```

Comment:
nit data isn't changing so `const` is appropriate

```suggestion
      const data = await UserSessions.findOne({
```